src/commandContext/replyPrivately.ts

```diff
@@ -9,6 +9,7 @@ import { replyWithPrivateMessage } from '../helpers/actions/messages/replyToMess
 export function replyPrivatelyFactory(
 	interaction: RepliableInteraction
 ): CommandContext['replyPrivately'] {
+	// eslint-disable-next-line @typescript-eslint/no-inferrable-types
```

Comment:
Maybe we should disable this rule outright. I quite like to be extra explicit with function argument types usually :3

Reply:
I'd still approve this PR if we don't, tho

Reply:
Good call